scripts/west_commands/runners/openocd.py

```diff
@@ -431,6 +447,9 @@ def do_attach_debug_rtt(self, command, **kwargs):
                 server_proc.terminate()
                 server_proc.wait()
 
+            if os_name != 'nt':
+                termios.tcsetattr(fd, termios.TCSAFLUSH, old_term)
```

Comment:
Shouldn't this be indented 1 more, as it has to be done in the `finally` case, as @mbolivar suggested?

Reply:
okey